### PR TITLE
Update committee and sponsors

### PR DESCRIPTION
**Pull request by CameronAavik**
Haven't actually tested this because I don't have a way to run Jekyll locally atm, need someone to quickly look it over to make sure it's all fine before it gets merged in.

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -18,63 +18,69 @@ We develop, hack and ultimately have fun.
...
     </a>
     </div>
     <div class="col s12 m6 l2  push-l1">
         <a href="https://www.tanda.co">
         <img class="sponsor-img" src="/img/sponsors/tanda.png">
     </a>
     </div>
+    <div class="col s12 m6 l2  push-l1">
+        <a href="https://www.technologyonecorp.com/">
+        <img class="sponsor-img" src="/img/sponsors/TechnologyOne.jpg>
```
Comment (wisebaldone) [inline about.md]: src is missing closing "

Comment (wisebaldone) [inline about.md]: {{site.baseurl}}/img/sponsors/TechnologyOne.jpg could also be used and keeping filenames all lowercase helps if its on a non case sensitive file system but no biggy.

Comment (CameronAavik) [inline about.md]: rip, thanks for spotting. will fix

**Comment by wisebaldone** [comment]
LGTM

Note: not built locally, but checked by inspection

**Comment by lewisjb** [comment]
cshield is now cat

**Comment by TRManderson** [comment]
Merging without checking, I trust you guys

**Comment by CameronAavik** [comment]
I don't trust me, but ok

**Comment by draganmarjanovic** [comment]
Was gonna test in the morning lel

**Comment by draganmarjanovic** [comment]
image

**Comment by CameronAavik** [comment]
As discussed in slack, needs a bit of padding between the two.

File: about.md (hunk 1 above)
Comment (JosephGarrone) [inline about.md]: I was pinged

**Pull request merged**